doc/api/nmo-activetasks.md

```diff
@@ -0,0 +1,15 @@
+nmo-activetasks(3) -- activetasks
+==============================
+
+## SYNOPSIS
+
+    nmo.commands.activetasks.getActiveTask(url, filter)
+
+
+## DESCRIPTION
+
+Get the list of active tasks from a specified url and filter it according to
+the supplied filter.
+
+The filter can be either the task type e.g replication, view_indexer or a
+database name found in either the source, target or database field.
```

Comment:
I'm not quite sure what you mean? Its basically a very basic filter that either filters on type of active task or on a database name found in either source, target or database field.